kmath-trajectory/src/commonMain/kotlin/space/kscience/kmath/trajectory/segments/Segment.kt

```diff
@@ -0,0 +1,5 @@
+package space.kscience.kmath.trajectory.segments
+
+public interface Segment {
+    public val length: Double
```

Comment:
Based on our research, segment "cost" is frequently not "static", but is computed based on the external context. We will think about that in future. Probably does not make sense to include it now.